config/routes.rb

```diff
@@ -5,6 +5,7 @@
   # You can have the root of your site routed with "root"
   root 'home#index'
   resources :cards
+  post 'trainer' => 'home#trainer'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.